wgpu-types/src/instance.rs

```diff
@@ -309,6 +309,8 @@ impl GlBackendOptions {
 pub struct Dx12BackendOptions {
     /// Which DX12 shader compiler to use.
     pub shader_compiler: Dx12Compiler,
+    /// Whether to use DirectComposition for managing presentation.
+    pub use_dcomp: bool,
```

Comment:
I think we should have an enum here (which will also allow us to have a `from_env`). We could call it `Dx12PresentationSystem` and have `DXGI` and `DirectComposition` as options